src/openepd/api/average_dataset/generic_estimate_sync_api.py

```diff
@@ -57,26 +57,33 @@ def get_by_openxpd_uuid(
 
     @overload
     def post_with_refs(
-        self, ge: GenericEstimateWithDeps, with_response: Literal[True]
+        self, ge: GenericEstimateWithDeps, with_response: Literal[True], exclude_defaults: bool = True
     ) -> tuple[GenericEstimate, Response]: ...
 
     @overload
-    def post_with_refs(self, ge: GenericEstimateWithDeps, with_response: Literal[False] = False) -> GenericEstimate: ...
+    def post_with_refs(
+        self, ge: GenericEstimateWithDeps, with_response: Literal[False] = False, exclude_defaults: bool = True
+    ) -> GenericEstimate: ...
 
     def post_with_refs(
-        self, ge: GenericEstimateWithDeps, with_response: bool = False
+        self, ge: GenericEstimateWithDeps, with_response: bool = False, exclude_defaults: bool = True
```

Comment:
Why we do this only for generic estimates, what about epds?

Reply:
My mistake, I'll update

Reply:
updated